src/client/CMakeLists.txt

```diff
@@ -53,7 +51,16 @@ elseif(PYPRT_LINUX)
 	target_link_libraries(${CLIENT_TARGET} PRIVATE stdc++fs)
 
 elseif(PYPRT_MACOS)
-	# TODO
+	target_compile_options(${CLIENT_TARGET} PRIVATE
+			-stdlib=libc++
+			-fvisibility=hidden -fvisibility-inlines-hidden
+			-Wall -Wextra -Wunused-parameter)
+
+	set_target_properties(${CLIENT_TARGET} PROPERTIES
+			INSTALL_RPATH "@loader_path"
+			INSTALL_RPATH_USE_LINK_PATH FALSE
+			SKIP_RPATH FALSE
```

Comment:
Any reason why you are explicitly setting SKIP_RPATH to FALSE here? Seems a bit redundant to me.